### ISSUE TITLE
migrate fails with (1146, "Table 'db.auth_group' doesn't exist")

### ISSUE THREAD

**Issue by matthewhegarty** [computer-lab/django-rest-framework-roles#6]
I have a ViewSet which references RoleViewSetMixin:

```
class MyViewSet(RoleViewSetMixin):
```

When I try to run migrate for a new installation, the process crashes with:

```
django.db.utils.ProgrammingError: (1146, "Table 'db.auth_group' doesn't exist")
```
[drfr_err.txt](https://github.com/computer-lab/django-rest-framework-roles/files/3043881/drfr_err.txt)

This is because the `RoleViewSetMixin` class contains a static call which tries to read from models which are not present (because migrate hasn't been run):

```
DEFAULT_GROUPS = [group.name.lower() for group in Group.objects.all()]
```
The solution is to move that initialisation to a constructor (something like):
```python
class RoleViewSetMixin(object):
    """A ViewSet mixin that parameterizes DRF methods over roles"""

    _viewset_method_registry = set(getattr(settings, "VIEWSET_METHOD_REGISTRY", DEFAULT_REGISTRY))

    def __init__(self):
        groups = [group.name.lower() for group in Group.objects.all()]
        self._role_groups = set(getattr(settings, "ROLE_GROUPS", groups))
```
What are your thoughts?  Happy to submit a PR if required.

There is a workaround which is to uncomment all paths from `urls.py`, run `manage.py migrate auth`, then re-add the URLs and run `manage.py migrate`

**Comment by r1b** [comment]
PRs welcome! I'm not really working with django these days but this seems like a straightforward fix.

**Comment by matthewhegarty** [comment]
[PR](https://github.com/computer-lab/django-rest-framework-roles/pull/7/files) (has been tested).

**Comment by r1b** [comment]
Closed by #7
